frontend/assets/javascripts/src/modules/analytics/facebook.js

```diff
@@ -1,6 +1,6 @@
 /* global fbq */
 /* eslint-disable no-unused-vars, no-underscore-dangle */
-define(['lodash/collection/forEach'], function(forEach) {
+define(['lodash/forEach'], function(forEach) {
```

Comment:
We updated `lodash` to the version `4.17.4` which changed where the functions are located (https://github.com/lodash/lodash/tree/amd).